packages/components/src/Text/Text.spec.tsx

```diff
@@ -34,6 +34,7 @@ describe("<Text />", () => {
     textRef.current.focus();
 
     const textElement = screen.getByText("Some Text");
+    // eslint-disable-next-line testing-library/no-node-access
```

Comment:
The new eslint rules don't like directly accessing `document.activeElement`, it seems 🤔 

We could disable the rule. Or upgrade testing library and get access to `toHaveFocus`.

Reply:
Or is the issue that we don't have `@testing-library/jest-dom` installed?

Reply:
That's probably it. I assumed we had that installed. But if not, that's definitely the problem.